autobuild/startup_blockcreator.toml

```diff
@@ -0,0 +1,45 @@
+[startup.tfchaind]
+name = "core.system"
+
+
+[startup.tfchaind.args[startup.tfchaind]
+name = "core.system"
+
+
+[startup.tfchaind.args]
+name = "/bin/tfchaind"
+# should use default values here?
+args = ["--rpc-addr={TFCHAIND_RPC_ADDR}", "--api-addr={TFCHAIND_APIADDR}", "--persistent-directory", "/mnt/data " , "--network={TFCHAIND_NETWORK}"]
+
+
+[startup.tfchainc]
```

Comment:
I assume this will try to start tfchainc as a long running process?

Reply:
indeed, this is not needed.
Depending how we want to approach the management of tfchaind, we might even not need tfchainc in the flist at all.
But for sure we don't need to start it at container creation

Reply:
removed it